frontend/src/components/Banner.js

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+
+import React from "react";
+import HeaderTypography from "./HeaderTypography";
+import ParagraphTypography from "./ParagraphTypography";
+import { Link } from "react-router-dom";
+import RoundedEdgesButton from "./RoundedEdgesButton";
+
+import { textColor, activeTextColor } from "../assets/jss/colorPalette";
+import bannerImage from "../assets/img/banner.png";
+
+export default function WelcomeBanner({ user }) {
+	const welcomeTitle = "Welcome to NFT The World!";
+	const welcomeSubtitle = <>
+		Breathtaking NFT shopping lounge.<br/>
+		Own your virtual map area from Nuremburg and Riga!
+	</>;
+	return <Banner user={user} title={welcomeTitle}>{ welcomeSubtitle }</Banner>;
+}
+
+/**
+ * Displays a welcoming background image with a button.
+ * @param user user object of the currently logged in account
+ * @returns {JSX.Element}
+ */
+export function Banner({ children, title, user }) {
+	const bannerTitle = <BannerTitle>{ title }</BannerTitle>;
+	const subTitle = <BannerSubTitle>{ children }</BannerSubTitle>;
+	const button = <BannerButton isLoggedIn={(user != null)} style={{ fontSize: "17px", }}/>;
```

Comment:
I see no advantage of defining these varialbes if we could just include them in the component. Also, you could just include the fontStyle style in the button component itself as we only use that component once anyways.

Reply:
Yes, both is legitimate. In my case, I aimed at easy maintainability, that means easier to change because who knows the future ;-) ? Also main reasons basically are lower indentation width and more levels of detail. But honestly I don't know how useful it really is in this case.